.pre-commit-config.yaml

```diff
@@ -181,6 +181,11 @@ repos:
       - id: prettier
         exclude: ^(packages/grid/helm|packages/grid/frontend/pnpm-lock.yaml|packages/syft/tests/mongomock)
 
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.16.0
+    hooks:
+      - id: pyupgrade
+
```

Comment:
We already enabled pyupgrade rules in ruff so this is redundant. We specifically don't run automated linting on `tests/mongomock` because it's a fork.

Reply:
reverted